src/main/java/dk/cs/aau/huppaal/presentations/HUPPAALPresentation.java

```diff
@@ -331,20 +318,154 @@ private void initializeColorSelector() {
+            proc.destroy();
+            return;
+        }
+
+        // Else start the run configuration
+        new Thread(() -> {
+            try {
+                var rt = Runtime.getRuntime();
+                if(config.program.isEmpty())
+                    throw new Exception("No program to run in selected run configuration");
+                var dir = new File(config.executionDir);
+                if(!(dir.exists() && dir.isDirectory()))
+                    throw new Exception(String.format("'%s' does not exist or is not a directory", config.executionDir));
+                proc = rt.exec(config.program + " " + config.arguments,
+                                ArrayUtils.merge(sysEnv, config.environmentVariables.split(";")),
+                                dir);
+                var stdi = new BufferedReader(new InputStreamReader(proc.getInputStream()));
+                var stde = new BufferedReader(new InputStreamReader(proc.getErrorStream()));
+                controller.runConfigurationExecuteButtonIcon.setIconLiteral("gmi-stop");
+                controller.runConfigurationExecuteButtonIcon.setIconColor(javafx.scene.paint.Color.web("#ff7e79"));
+                String s;
+                while((s = stdi.readLine()) != null) {
+                    var finalS = s;
+                    Platform.runLater(() -> CodeAnalysis.addMessage(finalS));
+                }
+                while((s = stde.readLine()) != null) {
+                    var finalS = s;
+                    Platform.runLater(() -> CodeAnalysis.addMessage(new CodeAnalysis.Message(finalS, CodeAnalysis.MessageType.ERROR)));
+                }
+                HUPPAAL.showToast(config.name + " finished("+proc.exitValue()+")");
+            } catch (Exception e) {
+                HUPPAAL.showToast(e.getMessage());
+                e.printStackTrace();
+            } finally {
+                controller.runConfigurationExecuteButtonIcon.setIconLiteral("gmi-play-arrow");
+                controller.runConfigurationExecuteButtonIcon.setIconColor(javafx.scene.paint.Color.WHITE);
+            }
+        }).start();
+    }
+
+    private void initializeRunConfigPicker() {
+        var runConfigsJson = HUPPAAL.preferences.get(RunConfigurationPreferencesKeys.ConfigurationsList, "[]");
+        var lastRunConfig = HUPPAAL.preferences.get(RunConfigurationPreferencesKeys.CurrentlySelected, "");
+        var runConfigurations = parseRunConfigurationsClearPreferencesIfFails(runConfigsJson);
+
+        // Add all the default things
+        controller.runConfigurationPicker.getItems().clear(); // Clear, because this function might be called again during runtime
+        controller.runConfigurationPicker.getItems().add(generateRunConfigurationEditButton());
+        if (runConfigurations.isEmpty())
+            controller.runConfigurationPicker.getItems().add(new RunConfigurationButton(Optional.empty(), new JFXButton("<no run configurations>")));
```

Comment:
Can this string be shown as the default value for the dropdown?

Reply:
Fixed in 69b536fbcdcf24112773004796a5e97e20718c47